wiki/Beatmap/Pattern/Mania/Chords/en.md

```diff
@@ -0,0 +1,33 @@
+---
+stub: true
+tags:
+  - chords
+---
+
+# Chords
+
+A **chord** is two or more notes that are hit at the same time. They are known to increase density and highlight different sounds within the music.
+
+## Jumps
+
+Also known as doubles, **jumps** are two notes pressed at the same  time. This is the most common chord type in 4-key osu!mania.
+
+![](img/jump.png)
+
+## Hands
+
+Also known as triples, **hands** are three notes pressed at the same time.
```

Comment:
Similar annotation here than on the jump. Hands are also called like this due to the need of using your hand to hit a triple in DDR.

Reply:
Same as https://github.com/ppy/osu-wiki/pull/8595/files#r1103866252

Reply:
Same as https://github.com/ppy/osu-wiki/pull/8595/files/281d7cd8babce6ebc8057bc8f409eaa728422c68#r1106892655